rosbag2_cpp/CMakeLists.txt

```diff
@@ -87,16 +87,19 @@ install(
 
 install(
   TARGETS ${PROJECT_NAME}
+  EXPORT export_${PROJECT_NAME}
   ARCHIVE DESTINATION lib
   LIBRARY DESTINATION lib
   RUNTIME DESTINATION bin)
 
 ament_export_include_directories(include)
 ament_export_libraries(${PROJECT_NAME})
+ament_export_targets(export_${PROJECT_NAME})
 ament_export_dependencies(pluginlib
   rosbag2_storage
   rosidl_runtime_c
   rosidl_runtime_cpp
+  rosidl_typesupport_cpp
```

Comment:
How is this related to the change?

Reply:
From my understanding, when importing a cmake target, all dependencies have to present at cmake's generation step as opposed to using `${<pkg>_LIBRARIES}` which is applied during linking. It's therefore important to export all necessary dependencies.